tests/cases/focus/keyboard_focus2.slint

```diff
@@ -0,0 +1,132 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+// cSpell: ignore backtab
+
+import { VerticalBox, ListView } from "std-widgets.slint";
+
+TestCase := Window {
+  width: 300px;
```

Comment:
```suggestion
  width: 300px;
  height: 300px;
```

The C++ test fails on the CI because the CI sets the native style. 
The native ListView doesn't have a minimum size, so the size of he window will be so that the ListView is 0,  and because the Flickable in it clips, the runtime will think this is a clipped thing and it will not focus it.

By setting an actual size to the window, we fix that problem.